src/main/java/com/controller/MoviesController.java

```diff
@@ -0,0 +1,23 @@
+package com.controller;
+
+import com.model.movies.Movies;
+import com.service.MoviesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/simple-movies-db", produces = MediaType.APPLICATION_JSON_VALUE)
+public class MoviesController {
+
+    @Autowired
+    private MoviesService moviesService;
+
+    @GetMapping(path = "/getAllMovies")
```

Comment:
It may be worth looking into the principles of REST - this path doesn't look quite right for a RESTful API

Reply:
changed it from getAllMovies to movies according to REST best practices